Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM busybox
+
+RUN dir=`mktemp -d` && trap 'rm -rf "$dir"' EXIT && \
+    wget -O- http://nixos.org/releases/nix/nix-1.7/nix-1.7-x86_64-linux.tar.bz2  | bzcat | tar x -C $dir && \
```

Comment:
I don't think that's even proper way to run nix with docker, I'm working on
service abstraction layer that would make declarative docker containers
possible.
On Jan 23, 2015 11:53 PM, "Wout Mertens" <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/NixOS/nixpkgs/pull/4642#discussion_r23486704>:
>
> > @@ -0,0 +1,12 @@
> > +FROM busybox
> > +
> > +RUN dir=`mktemp -d` && trap 'rm -rf "$dir"' EXIT && \
> > +    wget -O- http://nixos.org/releases/nix/nix-1.7/nix-1.7-x86_64-linux.tar.bz2  | bzcat | tar x -C $dir && \
>
> This needs to be updated for Nix 1.8. Also, why not simply use the install
> procedure curl https://nixos.org/nix/install | sh?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/4642/files#r23486704>.
>